### ISSUE TITLE
Top button bar disappeared

### ISSUE THREAD

**Issue by jpawz** [Leszek111/ReLaunchX#3]
Boyue T63 JDRead (Android 4.2.2).
Top button bar is obscured by android top black bar (this one with time, battery etc.) after fullscreen reader app close (Alreader or Orion Viewer in my case). This happens with favorites and recent opend windows. This situation didn't appear when files are opend from search result window.

**Comment by Leszek111** [comment]
Question to better understand/describe the bug

How is ReLaunchX configured? : Advanced > Appearance > Full Screen
The default is to always show the Android status bar (the one which obscure ReLaunchX view in your case). 
 
According to your description assuming the default settings, which always restore Android status bar, the app seems to work correctly when it is invoked back after full screen app as it restore the Android status bar but somehow does not re-adjust the app UI boundaries - sounds more like a system problem than the app.

**Comment by jpawz** [comment]
> How is ReLaunchX configured? : Advanced > Appearance > Full Screen

Full screen is unchecked.

Maybe it's system problem, but strange thing is when all three screens: "Last opened", "Favorites" and "Search results" use the same layout, the problem didn't occurs with the last one.
